platform/ios/missing_icon.json

```diff
@@ -0,0 +1,52 @@
+// 20190423145312
```

Comment:
I know you already removed this junk, but the file itself should also live in a more specific path.